recipe/meta.yaml

```
@@ -40,11 +42,11 @@ requirements:
     - llvmlite 0.40.0
     - numpy {{ numpy_min_ver }}
     - tbb-devel 2021.8.0
-    - _openmp_mutex            # [linux]
-    - importlib_metadata       # [py<39]
+    - _openmp_mutex 5.1        # [linux]
+    - importlib_metadata 6.0.0 # [py<39]
   run:
     - python
-    - importlib_metadata       # [py<39]
+    - importlib_metadata 6.0.0 # [py<39]
```

```suggestion
    - importlib_metadata  # [py<39]
```
See https://github.com/numba/numba/blob/04e81073b2c1e3ff4afa1da8513738e5e136775b/setup.py#L372